src/components/form/RadioButton.tsx

```diff
@@ -21,6 +21,14 @@ export const RadioButton = ({ showAsCard = false, label, helpText, hideLabel, ..
       {helpText ? <HelpText title={getPlainTextFromNode(helpText)}>{helpText}</HelpText> : null}
     </div>
   );
+
+  useEffect(() => {
+    // Name attribute is not added by digdir design system
+    if (inputRef.current && rest.name) {
+      inputRef.current.name = rest.name;
+    }
+  }, [rest.name]);
```

Comment:
Instead of working around shortcomings in the design system in our own code, we should probably [improve this in the design system](https://github.com/digdir/designsystem). Having this code here requires us to maintain it, it does not benefit anyone else using the design system, and would overwrite (and possibly break things) in the future if the design system adds the name on their own.

Reply:
Okei should i create an issue? Can we contribute to their repository or do we wait until they fix it?

Reply:
You can create a PR there just as you can to this repo - it's all open source. 🙌

Reply:
Created a PR here: https://github.com/digdir/designsystem/pull/903